Userland/Applications/SystemMonitor/ProcessModel.cpp

```diff
@@ -313,6 +313,24 @@ GUI::Variant ProcessModel::data(const GUI::ModelIndex& index, GUI::ModelRole rol
     return {};
 }
 
+Vector<GUI::ModelIndex, 1> ProcessModel::matches(const StringView& searching, unsigned flags, const GUI::ModelIndex&)
```

Comment:
Please don't specify an inline capacity unless you have a good reason to.

Reply:
I understand and agree with the premise, but that pattern is literally in every other place related to in-table highlighted search. I believe it should be addressed (globally) with a separate PR, since this one is already big enough.

Reply:
I'd disagree that the PR is already big enough. (I myself have submitted single commits with significantly larger changes.) It's commonplace to refactor related code to the one you're working on, even if it's slightly unrelated. Just put a "Refactor XYZ" commit first, then your functional changes. After all, you're already renaming a bunch of members for clarity, which is a good thing.

Reply:
While we're here, east const please :^)

Reply:
Huh. I guess one learns something new everyday. Sure then!